cli/config/configuration-fc.toml.in

```diff
@@ -222,8 +222,30 @@ path = "@SHIMPATH@"
 #enable_tracing = true
 
 [agent.@PROJECT_TYPE@]
-# There is no field for this section. The goal is only to be able to
-# specify which type of agent the user wants to use.
+# If enabled, make the agent display debug-level messages.
+# (default: disabled)
+#enable_debug = true
+
+# Enable agent tracing.
+#
+# If enabled, the default trace mode is "dynamic" and the
+# default trace type is "isolated". The trace mode and type are set
+# explicity with the `trace_type=` and `trace_mode=` options.
+#
+# Notes:
+#
+# - Tracing is ONLY enabled when `enable_tracing` is set: explicitly
+#   setting `trace_mode=` and/or `trace_type=` without setting `enable_tracing`
+#   will NOT activate agent tracing.
+#
+# - See https://github.com/kata-containers/agent/blob/master/TRACING.md for
+#   full details.
+#
+# (default: disabled)
+#enable_tracing = true
```

Comment:
I wonder if it's time to have a specific section for tracing, `[trace]` might be...

Reply:
This is a good point. Yes, we *could* make a sub-section like the one below, but it might involve more rework as none of the other sections have sub-sections in them:

```toml
[agent.kata]

enable_debug = true

[agent.kata.trace]
enabled = true
mode = "dynamic"
type = "isolated"
```

But we haven't done that historically. For example, we have two "groups" of options for the hypervisor section:

```
memory_slots = ...
memory_offset = ...

block_device_driver = ...
block_device_cache_set = ...
block_device_cache_noflush = ...
```

For consistency they would also need new subsections too.

I am tempted to suggest -- although not ideal I know -- that we keep the *status quo* for now and potentially totally rework the configuration handling in v2.